modules/ROOT/pages/limitations-self.adoc

```diff
@@ -1,21 +1,111 @@
-= Limitations for Runtime Fabric
+= Requirements and Limitations for Runtime Fabric
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Review the following table of limitations before installing, configuring, and managing Anypoint Runtime Fabric. 
+Review the following requirements and limitations before installing, configuring, and managing Anypoint Runtime Fabric. 
 
 [%header%autowidth.spread]
 |===
-| Limitation | Description | Enforced by Runtime Fabric
-| Node count | The maximum number of nodes is 400. | Recommended
-| Node types | VM-based nodes are required. For example, Fargate is not supported. | Yes
-| Replicas per application | The maximum number of replicas per application is 8. | Yes
-| Deployments per Runtime Fabric | The maximum number of deployments being managed by Runtime Fabric at any instance should not exceed 1600 for optimal performance. | Recommended
-| Associated environments per Runtime Fabric | You can associate a Runtime Fabric with up to 100 environments in any Business Group. For example, if you associate a development and a production environment with Org A and a dev environment with Org B, that is three environment associations. | Yes
-| Business groups | You can create up to 50 Runtime Fabrics per org in a Business Group. Any sub org can contain up to 50 Runtime Fabrics, in addition to any shared by another sub org. For example, if you have parent Org A and its child Org B, you can have 50 Runtime Fabrics in Org A and 50 in Org B. You can also share all 50 Runtime Fabrics from Org A with Org B, and as a result, you will see 100 Runtime Fabrics in total in the list view of Org B. | Yes
+| Limitation | Description 
+| Node types | VM-based nodes are required. For example, Fargate is not supported. 
+| Replicas per application | The maximum number of replicas per application is 8. 
+| Associated environments per Runtime Fabric | You can associate a Runtime Fabric with up to 100 environments in any Business Group. For example, if you associate a development and a production environment with Org A and a dev environment with Org B, that is three environment associations. 
+| Business groups | You can create up to 50 Runtime Fabrics per org in a Business Group. Any sub org can contain up to 50 Runtime Fabrics, in addition to any shared by another sub org. For example, if you have parent Org A and its child Org B, you can have 50 Runtime Fabrics in Org A and 50 in Org B. You can also share all 50 Runtime Fabrics from Org A with Org B, and as a result, you will see 100 Runtime Fabrics in total in the list view of Org B. 
```

Comment:
```suggestion
| Business groups | You can create up to 50 Runtime Fabrics per org in a Business Group. Any sub org can contain up to 50 Runtime Fabrics, in addition to any shared with another sub org. For example, for a parent Org A and its child Org B, you can have 50 Runtime Fabrics in Org A and 50 in Org B. You can also share all 50 Runtime Fabrics from Org A with Org B, and as a result, you will see 100 Runtime Fabrics in total in the list view of Org B. 
```
I'm not super certain about the use of abbreviations here - are `org` and `sub org` fine in this context? While `Org A` or better yet `OrgA` can be considered a name of the organisational unit the former two fall out of line, sort of.